CHANGELOG.md

```diff
@@ -8,6 +8,380 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+- `WordPress.Security.EscapeOutput` now ignores `exit`/`die` statements without a status being passed, preventing false positives on code after the statement.
+- `WordPress.Security.EscapeOutput` now has improved recognition that `print` can also be used as an expression, not only as a statement.
+- `WordPress.Security.EscapeOutput` now has much, much, much more accurate handling of code involving ternary expressions and should now correctly ignore the ternary condition in all long ternaries being examined.
+- `WordPress.Security.EscapeOutput` no longer disregards the ternary condition in a short ternary.
+- `WordPress.Security.EscapeOutput` no longer skips over a constant or property mirroring the name of one of the (auto-)escaping/formatting functions being targetted.
+- `WordPress.Security.EscapeOutput` no longer throws false positives for `*::class`, which will always evaluate to a plain string.
+- `WordPress.Security.EscapeOutput` no longer throws false positives on output generating keywords encountered in an inline expression.
+- `WordPress.Security.EscapeOutput` no longer throws false positives on parameters passed to `_e()` or `_ex()`, which won't be used in the output.
+- `WordPress.Security.EscapeOutput` no longer throws false positives on heredocs not using interpolation.
+- `WordPress.Security.NonceVerification` now respects that function names in PHP are case-insensitive when checking whether an array comparison function is being used.
+- `WordPress.Security.NonceVerification` now also checks for nonce verification when the `$_FILES` superglobal is being used.
+- `WordPress.Security.NonceVerification` now ignores properties named after superglobals.
+- `WordPress.Security.NonceVerification` now ignores list assignments to superglobals.
+- `WordPress.Security.NonceVerification` now ignores superglobals being unset.
+- `WordPress.Security.ValidatedSanitizedInput` now respects that function names in PHP are case-insensitive when checking whether an array comparison function is being used.
+- `WordPress.Security.ValidatedSanitizedInput` now respects that function names in PHP are case-insensitive when checking whether a variable is being validated using `[array_]key_exists()`.
+- `WordPress.Security.ValidatedSanitizedInput` improved recognition of interpolated variables and expression in the text strings. This fixes some false negatives.
+- `WordPress.Security.ValidatedSanitizedInput` no longer incorrectly regards an `unset()` as variable validation.
+- `WordPress.Security.ValidatedSanitizedInput` no longer incorrectly regards validation in a nested scope as validation which applies to the superglobal being examined.
+- `WordPress.WP.AlternativeFunctions` could previously get confused when it encountered comments in unexpected places.
+- `WordPress.WP.AlternativeFunctions` now correctly takes the `minimum_wp_version` into account when determining whether a call to `parse_url()` can switch over to using `wp_parse_url()`.
```

Comment:
```suggestion
- `WordPress.WP.AlternativeFunctions` now correctly takes the `minimum_wp_version` into account when determining whether a call to `parse_url()` should switch over to using `wp_parse_url()`.
```

Reply:
I think "can" is the right word here as it is about the minimum version check. Whether the function _should_ switch over may take more things into account.

Maybe "could" would be the best option ?

Reply:
I can go with `could`.